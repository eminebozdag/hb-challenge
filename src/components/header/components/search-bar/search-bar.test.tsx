import { render } from '@testing-library/react';
import React from 'react';
import SearchBar from './search-bar';

describe('<SearchBar />', () => {
   it('should render successfully', () => {
      // Arrange

      // Act
      const { getByTestId, container } = render(<SearchBar />);

      // Assert
      expect(getByTestId('search-input')).toBeDefined();
      expect(container.innerHTML).toMatchSnapshot();
   });
});
