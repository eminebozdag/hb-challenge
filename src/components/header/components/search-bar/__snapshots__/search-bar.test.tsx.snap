// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<SearchBar /> should render successfully 1`] = `"<div class=\\"search-box\\"><div class=\\"search-box__icon\\"><img alt=\\"search\\"></div><input data-testid=\\"search-input\\" placeholder=\\"25 milyon'dan fazla ürün içerisinde ara\\" class=\\"search-box__input\\"></div>"`;
